testing/api/test_qc_parallel.py

```diff
@@ -17,16 +19,23 @@ def gen_qc(path_qc):
 
 def test_many_qc():
     """Test many qc images can be made in parallel"""
+    # Turn on debug logging for fs.py to check mutex acquire/release times
+    # To check this output locally, run: `sct_testing -o log_cli=true testing/api/test_qc_parallel.py::test_many_qc`
+    screen_handler = logging.StreamHandler(stream=sys.stderr)
```

Comment:
Note: Setting `stream=sys.stderr` has the helpful effect of logging all `portalocker` DEBUG messages together at the end of the test output.

However... huh. I'm realizing that this behavior only occurs inside of PyCharm, and not more generally in terminals. So, this might be unnecessary?

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/884a3ad5-921a-47e6-846b-3adce19feb7c)

Reply:
I'm also realizing that this log output doesn't do a great job at identifying which process the log messages belong to, nor does it give any sort of timestamp... :thinking:

Reply:
At least it confirms that the test processes are actually trying to lock the file at the same time?